product-scenarios/4-single-sign-on/4.4-application-management/4.4.1-soap-apis/src/test/java/org/wso2/identity/scenarios/test/ApplicationManagementTestCase.java

```diff
@@ -470,7 +472,7 @@ public void testUpdateRoles() {
 
             RoleMapping mapping = new RoleMapping();
             LocalRole localRole = new LocalRole();
-            localRole.setLocalRoleName(IDP_ROLE_1);
+            localRole.setLocalRoleName("Internal/admin");
```

Comment:
use constant for `"Internal/admin"`

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/1b4ab737ecfa0d7092d2c697a13b8ad39d9ad538